### PR TITLE
PO Extractor - allow to import multi-lines msgstr

### PR DESCRIPTION
**Pull request by oliver-kriska**
this change should support when POT files contain multiline strings as msgstr
for example:
```
msgid "Hello"
msgstr "Hello string"

msgid "Multiline"
msgstr "Multiline string \n"
"Another line"

```

### CODE REVIEW

**Review by vi0dine — APPROVED** [review]

**Comment by oliver-kriska** [comment]
thanks @vi0dine, fyi I don't have write access so I can not merge it.

**Pull request merged**
